reconcile/utils/ocm.py

```diff
@@ -823,13 +830,41 @@ def _init_addons(self):
         api = f"{CS_API_BASE}/v1/addons"
         self.addons = self._get_json(api).get("items")
 
+    def _init_version_gates(self):
+        """Returns a list of version gates"""
+        api = f"{CS_API_BASE}/v1/version_gates"
+        self.version_gates = self._get_json(api).get("items")
+
     def get_addon(self, id):
         for addon in self.addons:
             addon_id = addon["id"]
             if id == addon_id:
                 return addon
         return None
 
+    def get_version_gates(self, version_prefix: str) -> list[Mapping[str, Union[str, bool]]]:
+        if not self.init_version_gates:
```

Comment:
`if not self.init_version_gates and not self.version_gates` , so we avoid getting the info several times when `self.init_version_gates = False` ?